### DISCUSSION TITLE
how to setup nginx with directus? found the issue

### DISCUSSION THREAD

**Discussion by Nisthar** [directus/directus#14567] [Archive]
So in my local pc, if you try to access directus. It sends two types of requests for loading js files.

one looks like `http://localhost:8055/admin/somejsfile.js`

the other looks like `http://localhost:8055/extensions/somefile.js` and it loads fine on localhost without using nginx.

So in a nginx server, you usually add a location block for `/admin` and rewrite every urls to proxy your `https://mysite.com/admin` which causes the issue.

when you do this, only the urls that matches `https://yourdomain/admin` are proxied to http://localhost:8055/admin
the other urls fails. so all other asset files like `/extensions/operations/index.js` fails 

TLDR: even though i put my `PUBLIC_URL` to `https://mysite.com/admin`, the request to some scripts are sent to `https://mysite.com/extensions/operations/index.js` instead of `https://mysite.com/admin/extensions/operations/index.js`

**Comment by Nisthar** [comment]
@rijkvanzanten I think this is an issue. can you check it?

**Reply by rijkvanzanten** [reply]
I don't think so, the correct route in the API is `/extensions/<type>/index.js`, so that looks good to me 👍🏻

**Reply by Nisthar** [reply]
> I don't think so, the correct route in the API is `/extensions/<type>/index.js`, so that looks good to me 👍🏻

Yeah the issue is my directus app is hosted at `https://mysite.com/admin` so its not able to load any scripts `https://mysite.com/extensions/index.js`

**Comment by Nisthar** [comment]
I managed to deploy using nginx. But i couldn't get https://mysite.com/admin to work.

I had to use a new subdomain https://admin.mysite.com/
